config/app_config.go

```diff
@@ -0,0 +1,104 @@
+package config
+
+import (
+	"os"
+	"strings"
+)
+
+type TelemetryConfig struct {
+	NewRelicAppName  string
+	NewRelicLicence  string
+	TelemetryEnabled bool
+}
+
+type ServerConfig struct {
+	APIPort                         string
+	PdfAPIURL                       string
+	Version                         string
+	MachineName                     string
+	ApplicationName                 string
+	BuildVersion                    string
+	Environment                     string
+	AppURL                          string
+	MaskLogs                        bool
+	TimeoutRegister                 string
+	TimeoutToken                    string
+	TimeoutDefault                  string
+	EnableRequestLog                bool
+	EnablePrintRequest              bool
+	ElasticURL                      string
+	TokenSafeDurationInMinutes      int
+	RetryNumberGetBoleto            int
+	InfluxDBHost                    string
+	InfluxDBPort                    string
+	RecoveryRobotExecutionEnabled   string
+	RecoveryRobotExecutionInMinutes string
+	EnableMetrics                   bool
+	EnableFileServerCertificate     bool
+	WaitSecondsRetentationLog       string
+	ForceTLS                        bool
+}
+
+type QueueConfig struct {
+	ConnQueue                        string
+	OriginExchange                   string
+	OriginQueue                      string
+	OriginRoutingKey                 string
+	TimeToRecoveryWithQueueInSeconds string
+	Heartbeat                        string
+	QueueMaxTLS                      string
+	QueueMinTLS                      string
+	QueueByPassCertificate           bool
+}
+
+func getServerConfig() ServerConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀 
```suggestion
func getServerConfig() *ServerConfig {
```